### PR TITLE
Use GTK themes

### LINKED ISSUE #245: [Enhancement] Respect system GTK theme
- [x] channel list text colors are extremely broken
- [x] chat input box (way too small, clips attachment icon)
- [x] hover effects look bad/should probably be disabled (might be a theme issue?)
- [x] attachment boxes are unpadded
- [x] embeds
- [x] reactions
- [ ] css color stuff for renderer
- [ ] ~~ScrolledWindow backgrounds~~
- [x] chat input size is weird
- [x] link colors
- [x] reaction border
- [x] connection boxes
- [x] attachment progress bar

You can test this on any system including Windows. See https://stackoverflow.com/questions/36464010/change-gtk3-look-on-windows

### PR DESCRIPTION
**Pull request by ouwou**
makes abaddon actually use the system theme
closes #245 

some examples. i think i want to use one as the default for windows but not sure which

abaddon_GwWKTTreng
abaddon_NQgy1QG9Zx
abaddon_zJkUPrlxRL
abaddon_81axNeWBby
abaddon_emLHkEpUCQ
abaddon_goE3nqi7fb
abaddon_vrarAGKCpU
abaddon_h7ZsmaDRLH
abaddon_ZBxCn8zsfO

### CODE REVIEW

**Comment by TheMorc** [comment]
looks pretty nice in light mode, now only if GTK3 itself could distinguish the system theme (since the basic gtk3 demo doesn't have dark mode too, until I force it) 🙃 

image

**Comment by ouwou** [comment]
* application-low-priority.css removed
* unread indicator, expander color, mention badge background all use selected background color (can be configured in style section of settings)
* mention badge text color uses selected text color (can be configured in style section of settings)
* channel color setting removed
* link color removed
* set status dialog redone
* message upload progress bar hidden unless a message upload is in progress (may be redone later)
* voice setting window layout improved

**Comment by ouwou** [comment]
will probably merge in a couple days. if anyone thinks anything needs changing/general feedback lemme know
* i use selected background color as an accent color in channel list. idk if theres any better ideas but it seems to work the best out of the themes i tried
* nsfw channel color is always red instead of based on theme. maybe itd be better if it showed that # with a ⚠️ like in discord but id have to rewrite a bit more than id like to (at least insofar as getting this to a point where i feel ok merging it). but also red is probably fine (you can change it in abaddon.ini anyways)
* basically no themes except adwaita style the HdyTab class `.needs-attention` which is disappointing since thats how you can tell if a tab is for an unread channel 😾
* im planning to use Material-Black-Cherry as the "default" theme (ie what will be packaged in windows builds) pic:
image
